dotCMS/src/main/java/com/dotmarketing/db/listeners/RollbackListener.java

```diff
@@ -0,0 +1,49 @@
+package com.dotmarketing.db.listeners;
+
+public class RollbackListener implements DotListener {
+    private final String key;
+    private final Runnable listener;
+    public RollbackListener(DotListener listener) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'listener' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5639225)